crypto/bls_multisig.go

```diff
@@ -114,6 +118,9 @@ func AggregateBLSSignatures(sigs []Signature) (Signature, error) {
 // The order of the keys in the slice does not matter since the aggregation
 // is commutative. The slice should not be empty.
 // No check is performed on the input private keys.
+// Error returns:
+//  - invalidInputsError if no signatures are provided (sigs is empty) or
+//    some keys are not BLS12 381 keys
```

Comment:
fixing the signature typo and adding that the only check done is about the Go type (no cryptographic check done as stated above).
```suggestion
// Expected error returns:
//  - invalidInputsError if keys is empty or at least one key type is not BLS12-381.
```

Reply:
suggestion included in https://github.com/onflow/flow-go/pull/1830/commits/42a51d24cee91f35b355670231e72f7f9aec52c5